### ISSUE TITLE
Add way to suppress predicates/edges from graph

### ISSUE THREAD

**Issue by grtjn** [patrickmcelwee/ml-visjs-graph-ng#13]
The rest extension contains a filter for rdf:type and rdfs:label, but there could be more (app-specific) predicates that one might want to suppress..

**Comment by patrickmcelwee** [comment]
Agreed, I've long thought there should be some API for this in the default service (once we make that server-side). Until now, I've done it by overriding the default mlpm module adding predicates to the filters in the SPARQL.

**Comment by grtjn** [comment]
You probably mean client-side.. :)
